hello-1/conanfile.py

```diff
@@ -1,18 +1,21 @@
-from conans import ConanFile
+from conans import ConanFile, tools
 
 class HelloConan(ConanFile):
     name = "hello"
     version = "0.1"
     exports = "*"
     generators = "qmake"
+    settings = "os", "compiler", "arch", "build_type"
 
     def build(self):
-        self.run('qmake')
-        self.run("make")
+        self.run('qmake -tp vc')
+        vcvars = tools.vcvars_command(self.settings)
+        self.run("%s && msbuild hello.vcxproj /p:configuration=debug" % vcvars)
 
     def package(self):
         self.copy("*.h", dst="include")
-        self.copy("*.a", dst="lib")
+        self.copy("*.a", dst="lib", keep_path=False)
+        self.copy("*.lib", dst="lib", keep_path=False)
```

Comment:
Libraries are named differently in Win, with .lib

Also I added the keep_path=False, because in Win, artifacts are generated into subfolders like "debug", and it is easier to get rid of it.